byron/ledger/executable-spec/src/Ledger/Delegation.hs

```diff
@@ -207,7 +208,14 @@ data DState = DState
   { _dStateDelegationMap :: Bimap VKeyGenesis VKey
     -- | When was the last time each genesis key delegated.
   , _dStateLastDelegation :: Map VKeyGenesis Slot
-  } deriving (Eq, Show)
+  } deriving (Eq, Show, Generic)
+
+instance NoUnexpectedThunks DState where
```

Comment:
Maybe it'd be nice to comment on why the `Generic` derived implementation can't/shouldn't be used here.